arbos/tx_processor.go

```diff
@@ -192,6 +196,13 @@ func (p *TxProcessor) StartTxHook() (endTxNow bool, gasUsed uint64, err error, r
 			panic(err)
 		}
 
+		gasPriceRefund := arbmath.BigMulByUint(arbmath.BigSub(tx.GasFeeCap, basefee), tx.Gas)
+		gasPriceRefund = arbmath.BigMin(gasPriceRefund, availableRefund)
+		availableRefund.Sub(availableRefund, gasPriceRefund)
+		if err := util.TransferBalance(&from, &tx.FeeRefundAddr, gasPriceRefund, evm, scenario); err != nil {
+			return true, 0, err, nil
```

Comment:
a retryable ticket should be created at this point
```suggestion
			return true, 0, err, underlyingTx.Hash().Bytes()
```
but also should be unreachable because from balance should >= availableRefund